src/scripts/targetingex.ts

```diff
@@ -192,13 +192,45 @@ namespace Scripts {
                     if (arr.length > 0) {
                         result.gameObject(arr[0].Serial());
                     }
+                } else if (value.toLowerCase() === 'hover') {
+                    const statusBars = Shared.GetArray(GlobalEnum.customStatusBars, []);
```

Comment:
a pro ten hover udelat vlastni metodu kterou tu budes volat :) toto uz je silene dlouhe

Reply:
no ono obecne jakykoliv souvisejici kousek nad 10 radku se da 'pojmenovat' a dat do funkce, a pak kdyz ctes kod tak si reknes.. aha tady resi hover a jedes na dalsi if else.. Kdyz jsem tu psal ty kusy kodu pro sebe, tak jsem to neresil, ale slibuju ze nebudu delat funkce na vic jak 20 radku :)

Reply:
respektive i souvisejici kousek nad 5 radku )